front-end/src/components/CommentList.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import { Col, Button, ListGroup, ListGroupItem, ListGroupItemHeading, ListGroupItemText, Card, CardTitle, Container, Row } from 'reactstrap';
+import { UserLogin } from '../types/UserLogin';
+import CreateEditPostModal from '../components/CreateEditPostModal';
+import PostListItem from '../components/PostListItem';
+import { Post } from '../types/Post';
+import { Author } from '../types/Author';
+import { PostComment } from '../types/PostComment';
+import CommentListItem from './CommentListItem';
+import { AxiosWrapper } from '../helpers/AxiosWrapper';
+
+interface Props {
+  postId: string;
+  postAuthor: Author;
+  loggedInUser: UserLogin | undefined;
+}
+
+/**
+ * Post list component to show list of posts
+ * @param props 
+ */
+export default function CommentList(props: Props) {
+  console.log("COMMENT LIST")
+
+  const [commentList, setCommentList] = useState<PostComment[] | undefined>(undefined);
+
+  useEffect(() => {
+    AxiosWrapper.get(`${props.postAuthor.url}posts/${props.postId}/comments/`, props.loggedInUser).then((res: any) => {
+      // TODO: filters for friend posts?
+      console.log(res.data);
+      const comments: PostComment[] = res.data;
+      // Reverse the posts so that they are in order (from newest to oldest).
+      setCommentList(comments.reverse());
+    }).catch((err: any) => {
+      console.error(err);
+    });
+  }, []);
+
+  return (
+    <>
+      <Col>
```

Comment:
nit: It's a bit less idiomatic if we use `<Cols>` without `<Containers>` and `<Rows>`. e.g.
```tsx
<Container>
  <Col>
    <Row>
    </Row>
  </Col>
</Container>
```
I've found from past experience that using components in ways that they weren't exactly intended occasionally led to undesired css positioning and media queries that were hard to debug. I think it should be fine in this instance, but I think it would be good to use Reactstrap the way the framework intends to ensure that the CSS behaves consistently going forward

Reply:
put the `<Col>` inside a `<Container>`. Wrapping the cards/comment items in a `<Row>` made the components behave weird